dockstore-client/src/test/java/io/dockstore/client/cli/LaunchNoInternetTestIT.java

```diff
@@ -0,0 +1,304 @@
+package io.dockstore.client.cli;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+import io.dockstore.common.Utilities;
+import io.dropwizard.testing.ResourceHelpers;
+import org.apache.commons.io.FileUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.ExpectedSystemExit;
+import org.junit.contrib.java.lang.system.SystemErrRule;
+import org.junit.contrib.java.lang.system.SystemOutRule;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests CLI launching with image on filesystem instead of internet
+ * In general, ubuntu:0118999881999119725...3 is the file that exists only on filesystem and hopefully never on internet
+ * All tests will be trying to use that image
+ * @author gluu
+ * @since 1.6.0
+ */
+public class LaunchNoInternetTestIT {
+    @Rule
+    public final SystemOutRule systemOutRule = new SystemOutRule().enableLog();
+
+    @Rule
+    public final SystemErrRule systemErrRule = new SystemErrRule().enableLog();
+
+    @Rule
+    public final ExpectedSystemExit exit = ExpectedSystemExit.none();
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    public static String docker_images;
+    private static final File DESCRIPTOR_FILE = new File(ResourceHelpers.resourceFilePath("nonexistent_image/CWL/nonexistent_image.cwl"));
+    private static final File YAML_FILE = new File(ResourceHelpers.resourceFilePath("echo-job.yml"));
+    private static final String FAKE_IMAGE_NAME = "ubuntu:0118999881999119725...3";
+
+    /**
+     * Downloading an image with bash (Nextflow needs it) and saving it on the filesystem as something weird that is unlikely to be on the internet
+     * to make sure that the Dockstore CLI only uses the image from the filesystem
+     *
+     * @throws IOException
+     */
+    @BeforeClass
+    public static void downloadCustomDockerImage() throws IOException {
+        Utilities.executeCommand("docker pull ubuntu:latest", System.out, System.err);
```

Comment:
Ubuntu seems a bit big, how about the Bash image?
https://hub.docker.com/r/library/bash/

Reply:
Tried it, Nextflow it to work with:

$ docker run -it bash /bin/bash
/usr/local/bin/docker-entrypoint.sh: line 11: /bin/bash: No such file or directory

Reply:
Create a Dockerfile from that container above and add a symbolic link